js/authentication.md

```diff
@@ -1662,6 +1550,140 @@ Note: To work with Service Interface Objects, your Amazon Cognito users' [IAM ro
 For the complete API documentation for Authentication module, visit our [API Reference](https://aws-amplify.github.io/amplify-js/api/classes/authclass.html)
 {: .callout .callout--info}
 
+
+## Switching Authentication Flow Type
+
+For client side authentication flow, there are three different ways you can use:
+
+1. `USER_SRP_AUTH`: The `USER_SRP_AUTH` flow is used by default and utilizes the SRP protocol (Secure Remote Password) to encrypt the password on the client before it's sent to the back-end. It is a more secure way of sending user credentials over the network and is, therefore, the recommended approach.
```

Comment:
Add target="_blank" link to SRP protocol
https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol

password is not sent to the back-end so remove that. The password verifier is sent, password never leaves the client and is unknown to the server (check wiki link there).

Update to something like:

The `USER_SRP_AUTH` flow uses the [Secure Remote Password](https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol) where the password never leaves the client and is unknown to the server. This is the recommended flow.

Reply:
Done.